components/layout_2020/positioned.rs

```diff
@@ -22,7 +22,7 @@ use style::Zero;
 
 #[derive(Debug, Serialize)]
 pub(crate) struct AbsolutelyPositionedBox {
-    pub contents: IndependentFormattingContext,
```

Comment:
I’m not thrilled about this one (`layout_context` and `style_content` are groupings of transient data used "on the side" during layout, rather than the "object" of what we’re doing), but I don’t have a better suggestion.